docs/tags.rst

```diff
@@ -114,7 +114,8 @@ The following values are take from the ``BoundField``:
 - id_for_label
 - label
 - name
-- value
+- value (value of the field converted to string)
+- raw_value (e.g. True/False for BooleanField or datetime.datetime for DateTimeField)
```

Comment:
Oh, I wondered if you'd see string versions when handling a rerender after non-valid form data submitted.